### ISSUE TITLE
Crop Decimate Node: On Parameter Set

### ISSUE THREAD

**Issue by mhubii** [ros-perception/image_pipeline#967]
Thank you for providing this great package.

I believe dynamic parameters are not supported in  

https://github.com/ros-perception/image_pipeline/blob/rolling/image_proc/src/crop_decimate.cpp

**Comment by ahcorde** [comment]
There is no support for now, but happy to review a PR if you decide to contribute

**Comment by mhubii** [comment]
Would the PR aim at rolling?

**Comment by ahcorde** [comment]
@mhubii yes, and we can backport the changes later if the PR doesn´t break ABI or API
